cypress/e2e/HomePage/delete.project.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Test homepage: project deletion
+
+  Scenario: Delete existing project
+    Given I clear cache
+    And I visit the "/"
+    And I click on "[data-cy=\"new-project\"]"
+    And I expect current url is "modelizer/project-[a-f0-9]{8}/model"
+    Then I click on "[data-cy=\"app-logo-link\"]"
+    And I expect current url is "/"
+
+    When I click on "[data-cy=\"delete-project\"]"
+    And I click on "[data-cy=\"delete-project-form\"] [data-cy=\"confirm-delete-project-checkbox\"]"
+    And I click on "[data-cy=\"delete-project-form\"] [data-cy=\"delete-project-form-submit\"]"
+
+    Then I expect "positive" toast to appear with text "Project has been deleted 😥"
+    And I expect "[data-cy=\"delete-project-form\"]" is closed
+    And I expect localstorage field "projects" is '{}'
```

Comment:
do not check localstorage, but check that the deleted project card no longer exists

Reply:
done